tests/ml/test_ml_model_pytest.py

```diff
@@ -460,110 +471,3 @@ def test_lgbm_classifier_objectives_and_booster(
 
         # Clean up
         es_model.delete_model()
-
-    # If both overwrite and es_if_exists is given.
-    @requires_sklearn
-    @pytest.mark.parametrize("compress_model_definition", [True, False])
```

Comment:
Why are we removing these test cases?

Reply:
I removed 4 test cases
1st test => if both overwrite and `es_if_exists` parameters are given. I removed this because `es_if_exists` parameter is tested as part of other tests
2,3rd test => Just for the overwrite parameter. Since we are deprecating I removed these

4th test => I combined it with another test to make it efficient.

Let me know for any changes :)